go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/google/go-querystring v1.0.0 // indirect
 	github.com/motemen/go-loghttp v0.0.0-20170804080138-974ac5ceac27
 	github.com/motemen/go-nuts v0.0.0-20180315145558-42c35bdb11c2 // indirect
+	github.com/pkg/errors v0.8.1
```

Comment:
Curious about this added dependency 🤔 
Isn't https://golang.org/pkg/errors/ enough to cover this usage?

Reply:
good question - looking into this I ended up removing this, since we're already using "fmt" we may as well use fmt.Errorf here instead